content/posts/your-versions-your-urls.md

```diff
@@ -0,0 +1,21 @@
+title: Your versions, your URLs
+date: April 17, 2025
+description: Announcing the ability to customize the URLs your versions are served from.
+category: Feature announcement
+tags: hosting
+authors: Santos Gallegos
+status: published
+image:
+image_credit:
+
+Versions are a core part of Read the Docs,
+they let you host multiple versions of your documentation.
```

Comment:
```suggestion
Versions are a core part of Read the Docs.```

Reply:
I think you may have missed the rest of the suggestion